grpc_opa/authorizer.go

```diff
@@ -252,6 +258,8 @@ func (a *DefaultAuthorizer) Evaluate(ctx context.Context, fullMethod string, grp
 	}
 
 	if !opaResp.Allow() {
+		forbidReason := opaResp.rbacForbiddenReason()
+		logger.Infof("Request forbidden because these RBAC checks failed:%s", forbidReason)
```

Comment:
this will make it easier to find in kibana
```suggestion
		logger.WithField("msg", forbidReason).Info("obligation_forbidden msg")
```